src/QuLDE.jl

```diff
@@ -46,12 +47,92 @@ struct QuLDEParam{W,L,Q, VType <: GeneralMatrixBlock{1}, SType <: GeneralMatrixB
     end
 end
 
-function quldecircuit(blk::QuLDEParam, M::Matrix)
-    T = Int(log2(blk.k+1))
-    siz, = size(M)
-    nbit = Int(log2(siz))
+struct QuLDEnonUnitParam{W,L,Q,NL,Nbit, VType <: GeneralMatrixBlock{1}, SType <: GeneralMatrixBlock{Q}, LType <: GeneralMatrixBlock{NL}}
+    k::Int
+    t::L
+    l::Int
+    C_tilda::W
+    D_tilda::W
+    N::W
+    V::VType
+    VS1::SType
+    VS2::SType
+    WS1::SType
+    WS2::SType
+    VT::LType
+    WT::LType
+    F::Array{GeneralMatrixBlock{Nbit,Nbit,Complex{Float64},Array{Complex{Float64},2}},1}
+
+    function QuLDEnonUnitParam(k::Int,t::L,l::Int,prob::QuLDEProblem) where {L}
+        C(m) = norm(prob.u0)*((opnorm(prob.A)*t*2)^(m))/factorial(m)
+        D(n) = norm(prob.b)*(opnorm(prob.A)*t*2)^(n-1)*t/factorial(n)
+        nbit, = size(prob.u0)
+        nbit = Int(log2(nbit))
+        Mu = prob.A/opnorm(prob.A)
+        B1 = 1/2*(Mu + Mu')
+        B2 = -im/2*(Mu - Mu')
+        iden = Matrix{ComplexF64}(I,size(B1))
+        F = Array{GeneralMatrixBlock{nbit,nbit,Complex{Float64},Array{Complex{Float64},2}},1}(undef,4)
+        F[1] = matblock(B1 + im*sqrt(iden - B1*B1))
+        F[2] = matblock(B1 - im*sqrt(iden - B1*B1))
+        F[3] = matblock(im*B2 - sqrt(iden - B2*B2))
+        F[4] = matblock(im*B2 + sqrt(iden - B2*B2))
+        #tested
+
+        C_tilda = 0
+        D_tilda = 0
+        for i in 1:k
+            C_tilda = C_tilda + C(i)
+            D_tilda = D_tilda + D(i)
+        end
+        C_tilda = C_tilda + C(0)
+        N = sqrt(C_tilda^2 + D_tilda^2)
+        V = [C_tilda D_tilda; D_tilda -1*C_tilda]/N
```

Comment:
Why not use `ComplexF64[C_tilda D_tilda ... ]` directly? There is an extra conversion happens